models.rb

```diff
@@ -18,8 +18,10 @@ def validate
     errors.add(:name, 'cannot be empty') if !name || name.empty?
   end
 
-  def remove
-    projects.each(&:reassign_resources) unless projects.empty?
+  def remove(default_project)
```

Comment:
We can always look up the default project by name, so I am not sure we need this change - in reassign_resources, just ```Project.find(name: 'default')`` since I think it will only ever be used to reassign to the system default project, which should always have the name 'default'